MAUI/DataGrid/Column-types.md

```diff
@@ -337,6 +337,56 @@ dataGrid.Columns.Add( new DataGridTextColumn()
 {% endhighlight %}
 {% endtabs %}
 
+### Formatting DataGridTextColumn with different culture
+
+To apply different CultureInfo for DataGridColumns, use the `DataGridColumn.CultureInfo` property. Assign format of the string to this property. Based on the type of the property the column is associated to format the value. You can use different StringFormats to customize values displayed in the record cells.
```

Comment:
To apply a different `CultureInfo` for `DataGridColumns`, use the `DataGridColumn.CultureInfo` property. Assign the desired string format to this property. The column will format the value based on the type of the associated property. You can use different `StringFormats` to customize the values displayed in the record cells."

Reply:
Updated